packages/react-components/src/components/Dialpad/Dialpad.test.tsx

```diff
@@ -59,7 +60,7 @@ describe('Dialpad tests', () => {
 
   test('Clicking on dialpad button 6 should send the corresponding dtmf tone Num6', async () => {
     const testLocale = createTestLocale({
-      dialpad: { placeholderText: Math.random().toString() }
+      dialpad: { placeholderText: Math.random().toString(), deleteButtonAriaLabel: Math.random().toString() }
```

Comment:
is there not a string that we already localize for us to test with?

Reply:
This is for making sure string localization works. For example see CameraButton.test.tsx